package.json

```
@@ -23,7 +23,7 @@
     "test:production": "yarn workspace ember-data test:production",
     "test:try-one": "yarn workspace ember-data test:try-one",
     "test:docs": "yarn workspace ember-data test:docs",
-    "test:encapsulation": "yarn workspace serializer-encapsulation-test-app test",
+    "test:encapsulation": "lerna run --scope *-encapsulation-test-app test",
```

```suggestion
    "test:encapsulation": "lerna run --scope='*-encapsulation-test-app' test",
```

The quotes are necessary as otherwise some command environments will try to expand it eagerly with the `*` at the beginning like it is